### PR TITLE
feat: upgrade Fuel SDK to 0.94.8

### PR DESCRIPTION
**Pull request by arthurgeron**

### CODE REVIEW

File: packages/solana-connector/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "peerDependencies": {
     "@web3modal/core": ">=5.0.0",
     "@web3modal/scaffold": ">=5.0.0",
-    "fuels": ">=0.94.6"
+    "fuels": ">=0.94.98"
```
Comment (helciofranco) [inline packages/solana-connector/package.json:27]: there's a typo everywhere here.

```suggestion
    "fuels": ">=0.94.8"
```

**Review by helciofranco — CHANGES_REQUESTED** [review]

File: packages/solana-connector/package.json (hunk 1 above)
Comment (arthurgeron) [reply packages/solana-connector/package.json]: Nice catch, thanks

File: packages/connectors/package.json [hunk 2]
```diff
@@ -18,11 +18,12 @@
     "ts:check": "tsc --noEmit"
   },
   "peerDependencies": {
-    "fuels": ">=0.94.6"
+    "fuels": ">=0.94.8"
   },
   "dependencies": {
     "@ethereumjs/util": "9.0.3",
     "@ethersproject/bytes": "5.7.0",
+    "@fuels/connectors": "0.29.2",
```
Comment (helciofranco) [inline packages/connectors/package.json:26]: Should we add it?

**Review by helciofranco — DISMISSED** [review]

**Review by helciofranco — APPROVED** [review]

File: packages/evm-predicates/src/generated/predicates/index.ts [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
   generationDate as generationDate1725479113004,
 } from './0xfdac03fc617c264fa6f325fd6f4d2a5470bf44cfbd33bc11efb3bf8b7ee2e938';
 
 export const PREDICATE_VERSIONS = {
+  '0x65e5877f81ba8d5bf10d301b8096c7529d11d88d9cc5bbe566d2cc1010544390': {
+    predicate: { abi: abi1727718552630, bin: bin1727718552630 },
+    generatedAt: generationDate1727718552630,
+  },
   '0xfdac03fc617c264fa6f325fd6f4d2a5470bf44cfbd33bc11efb3bf8b7ee2e938': {
     predicate: { abi: abi1725479113004, bin: bin1725479113004 },
```
Comment (LuizAsFight) [inline packages/evm-predicates/src/generated/predicates/index.ts:21]: did the bytecode hash change? if not we dont need to generate a new version

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: packages/evm-predicates/src/generated/predicates/index.ts (hunk 3 above)
Comment (arthurgeron) [reply packages/evm-predicates/src/generated/predicates/index.ts]: Great point, it didn't change. I've reverted the predicate generation

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**